opa/views/session_views.xml

```diff
@@ -0,0 +1,133 @@
+                <field name="title"/>
+                <field name="course_id"/>
+            </calendar>
+        </field>
+    </record>
+
+    <record id="session_gantt_view" model="ir.ui.view">
+        <field name="name">session.gantt.view</field>
+        <field name="model">session</field>
+        <field name="arch" type="xml">
+            <gantt string="Sessions" date_start="starts_on" date_stop="ends_on"/>
+        </field>
+    </record>
+
+    <record id="session_graph_view" model="ir.ui.view">
+        <field name="name">session.graph.view</field>
+        <field name="model">session</field>
+        <field name="arch" type="xml">
+            <graph string="Attendees by Session" type="line">
+                <field name="course_id"/>
+                <field name="attendee_count" type="measure"/>
+            </graph>
+        </field>
+    </record>
+
+    <record id="session_kanban_view" model="ir.ui.view">
+        <field name="name">session.kanban.view</field>
+        <field name="model">session</field>
+        <field name="arch" type="xml">
+            <kanban default_group_by="course_id">
+                <field name="title"/>
+                <field name="course_id"/>
+                <field name="starts_on"/>
+                <field name="duration"/>
+                <templates>
+                    <t t-name="kanban-box">
+                        <div t-attf-class="oe_kanban_content oe_kanban_global_click">
+                            <div>
+                                <strong class="o_kanban_record_title">
+                                    <field name="title"/>
+                                </strong>
+                            </div>
+                            <div>
+                                <span class="o_kanban_record_subtitle">
+                                    <field name="course_id"/>
+                                </span>
+                            </div>
+                            <div class="o_kanban_record_bottom">
+                                <div class="oe_kanban_bottom_left">
+                                    <span>
+                                        Start: <field name="starts_on"/>
+                                    </span>
+
+                                </div>
+                                <div class="oe_kanban_bottom_right">
+                                    <span>
+                                        Duration: <field name="duration"/> (hrs)
+                                    </span>
+                                </div>
+                            </div>
+                        </div>
+                    </t>
+                </templates>
+            </kanban>
+        </field>
+    </record>
+
+    <record id="session_form_view" model="ir.ui.view">
+        <field name="name">session.form.view</field>
+        <field name="model">session</field>
+        <field name="arch" type="xml">
+            <form>
```

Comment:
Missing string?

Reply:
In a latter comment @frahikLV  said we should not add strings on forms.